### PR TITLE
Add meta-tag for header; Fixing problem of #24;

### PR DESCRIPTION
**Pull request by DNagorny**
Sorry for changes in formatting, please revert them if you want.
See #24.

### CODE REVIEW

**Comment by fmcarvalho** [comment]
Sorry, but I am in vacation right now and I will look at this issue just in September 1 st.

Thanks

File: src/main/java/htmlflow/elements/HtmlHead.java [hunk 1]
```diff
@@ -29,15 +29,34 @@
...
+    public HtmlScriptBlock<T> scriptBlock() {
+        return this.addChild(new HtmlScriptBlock<T>());
+    }
+    
+    public HtmlHead<T> linkCss(final String href) {
+        this.addChild(new HtmlLinkCss<T>(href));
+        return this;
+    }
+    
+    public HtmlHead<T> meta(final String attr, final String value) {
```
Comment (DNagorny) [inline src/main/java/htmlflow/elements/HtmlHead.java]: new meta-tag for header

**Comment by DNagorny** [comment]
@fmcarvalho 
Did you have a chance to look into this?

**Comment by fmcarvalho** [comment]
Not yet. I will do it this week.

**Comment by fmcarvalho** [comment]
@DNagorny  I already found the problem. It will be solved using the following statement to load the resource:

```java
HtmlView.class
   .getClassLoader()
   .getResource("templates/HtmlView-Header.txt");
```

I tested this fix in a WAR application depending of `HtmlFlow` jar and it worked fine.

I will include this fix on release 1.2 and I will try to publish it next Monday.

Thanks

**Comment by DNagorny** [comment]
Shall I remove my PR?

**Comment by fmcarvalho** [comment]
You can keep it. I will use your proposal of a meta tag and include it on release 1.2.

**Comment by DNagorny** [comment]
Ok, but I should've extended from HtmlSingleElement instead of WriterComposite, just as a remark. :)

**Pull request closed without merging**